javascript/react_js/react_testing_part_one.md

```diff
@@ -129,6 +131,23 @@ The tests speak for themselves. In the first test, we utilize snapshots to check
 
 It's also important to note that after every test, React Testing Library unmounts the rendered components. That's why we render for each test. For a lot of tests for a component, the `beforeEach` jest function could prove handy.
 
+Notice that the callback function for the second test is asynchronous. `user.click()` simulates the asynchronous nature of user interaction. 
+The testing library devs making their user-event APIs [asynchronous](https://github.com/testing-library/user-event/releases/tag/v14.0.0) is a recent feature. Some examples from other resources/tutorials might call `userEvent.click()` synchronously.
```

Comment:
I like that you link to the changelog, but can you rework this a bit so that the link text is more descriptive of what it's actually linking to?

Reply:
@wise-king-sullyman 
I'm not sure if it's just on my end, but the code block is not rendering properly on the [lesson page](https://www.theodinproject.com/lessons/node-path-javascript-react-testing-part-1).
![Screenshot from 2023-04-02 22-08-08](https://user-images.githubusercontent.com/31008330/229358092-6ee1351e-2108-42c9-9db3-65bd0b702ce2.png)

Although [GitHub's markdown preview](https://github.com/TheOdinProject/curriculum/blob/main/javascript/react_js/react_testing_part_one.md) seems to be okay.
![Screenshot from 2023-04-02 22-03-55](https://user-images.githubusercontent.com/31008330/229357872-1c3437c3-1631-4355-9ec5-0ac5c031ab4c.png)

Checking the raw md, it might be that I have indented the code block by accident. I have no idea how to test it on a webpage and only relied on GitHub's markdown preview.
Can you please check if this needs fixing. Thank you.

Reply:
Ah yes that's a bit of an issue, indentation does look like the problem. Do you have the time to make another quick PR fixing that today? If so please @ me in the opening comment of it so that I get a notification when it's up.

Also for future reference our [markdown preview tool](https://www.theodinproject.com/lessons/preview) parses things in the same way our site does so that you can check these kinds of things beforehand.